src/tests/ftest/server/metadata.py

```diff
@@ -65,7 +65,7 @@ class ObjectMetadata(TestWithServers):
     CREATED_CONTAINERS_MIN = 2900
 
     # Number of created containers that should not be possible
-    CREATED_CONTAINERS_LIMIT = 3500
+    CREATED_CONTAINERS_LIMIT = 7500
```

Comment:
Just pointing out this might be a moving target in the future

Reply:
Agreed. In some older discussions IIRC it was suggested to restructure the test to first probe for the number of create-able containers in a first pass, and then base any subsequent passes success criteria upon this discovered number (that there be some predictability e.g., as tested in the addremove test). Though I suppose there is some value in those constants in the current code do allow us to flag unexpected large changes such as this one. Thanks for the review & landing.